apps/ledger-live-desktop/src/renderer/screens/account/index.tsx

```diff
@@ -146,6 +147,17 @@ const AccountPage = ({
 
   const color = getCurrencyColor(currency, bgColor);
 
+  function isBitcoinBasedAccount(account: Account | AccountLike): account is BitcoinAccount {
+    return (account as BitcoinAccount).bitcoinResources !== undefined;
+  }
+
+  function isBitcoinAccount(account: BitcoinAccount): boolean {
+    return account.currency.id === "bitcoin";
+  }
```

Comment:
could be moved outside in an helper ? 

also you can just do `account: Account`

Reply:
Added to common lib so I will be able to use it on LLM too